src/GHC/Util.hs

```diff
@@ -0,0 +1,186 @@
+{-# LANGUAGE PackageImports #-}
+{-# OPTIONS_GHC -Wno-missing-fields #-}
+
+module GHC.Util (
+    dynFlags
+  , parseFileGhcLib
+  , ParseResult (..)
+  , pprErrMsgBagWithLoc
+  , getMessages
+  , SDoc
+  , Located
+  ) where
+
+import "ghc-lib-parser" HsSyn
+import "ghc-lib-parser" DynFlags
+import "ghc-lib-parser" Platform
+import "ghc-lib-parser" Fingerprint
+import "ghc-lib-parser" Config
+import "ghc-lib-parser" Lexer
+import "ghc-lib-parser" Parser
+import "ghc-lib-parser" SrcLoc
+import "ghc-lib-parser" FastString
+import "ghc-lib-parser" StringBuffer
+import "ghc-lib-parser" ErrUtils
+import "ghc-lib-parser" Outputable
+import "ghc-lib-parser" GHC.LanguageExtensions.Type
+
+import Data.List
+import Language.Preprocessor.Unlit
+
+-- See note [ghc-lib-parser directives] in 'HSE/All.hs'.
+{-# ANN module "HLint: ignore Avoid restricted extensions" #-}
+{-# ANN module "HLint: ignore Avoid restricted flags" #-}
+
+fakeSettings :: Settings
+fakeSettings = Settings
+  { sTargetPlatform=platform
+  , sPlatformConstants=platformConstants
+  , sProjectVersion=Config.cProjectVersion
+  , sProgramName="ghc"
+  , sOpt_P_fingerprint=Fingerprint.fingerprint0
+  }
+  where
+    platform =
+      Platform{platformWordSize=8
+              , platformOS=OSUnknown
+              , platformUnregisterised=True}
+    platformConstants =
+      PlatformConstants{pc_DYNAMIC_BY_DEFAULT=False,pc_WORD_SIZE=8}
+
+fakeLlvmConfig :: (LlvmTargets, LlvmPasses)
+fakeLlvmConfig = ([], [])
+
+enabledExtensions :: [Extension]
+enabledExtensions =
```

Comment:
Can we just do `[minBound..maxBound]` and exclude ones we don't like?

Reply:
going with `enabledExtensions = [Cpp .. StarIsType]` and will submit a patch for the `Bounded`.